ricecooker/classes/nodes.py

```diff
@@ -427,6 +427,26 @@ def truncate_fields(self):
 
         super(TreeNode, self).truncate_fields()
 
+    def sort_topic_nodes(self, channel, key = None, reverse = False):
+        """
+        Sort Topic Nodes in channel
+        :param channel: Channel to sort
+        :param key: A Function to execute to decide the order. Default None
+        :param reverse: A Boolean. False will sort ascending, True will sort descending. False by default
+        :return: Sorted channel
+        """
+        # default natural sorting
+        if not key:
+            convert = lambda text: int(text) if text.isdigit() else text.lower() 
+            key = lambda key: [ convert(re.sub(r'[^A-Za-z0-9]+', '', c.replace('&', 'and'))) for c in re.split('([0-9]+)', key.title) ]
+        try:
+            channel.children = sorted(channel.children, key = key, reverse = reverse)
+        except Exception as e:
```

Comment:
Please remove the exception part. If the developer wants the default sorting, they can write fallback code themselves.

Reply:
removed exception